programs/psa/psa_hash_demo.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+. "${0%/*}/../demo_common.sh"
+
+msg <<'EOF'
+This script demonstrates the use of the PSA cryptography interface to
```

Comment:
Not this script, just the program it runs.

```suggestion
This program demonstrates the use of the PSA cryptography interface to
```